edi_account_edifact_oca/components/validate.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2023
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.addons.component.core import Component
+
+
+class EDIStorageValidateComponent(Component):
+    _name = "edi.output.account.move.validate"
+    _inherit = "edi.component.validate.mixin"
+    _usage = "output.validate.account.move"
+
+    def validate(self, value):
```

Comment:
validation makes sense for xml files to validate using a xsd, but for edifact I don't think we need it